scripts/scil_tractogram_pairwise_comparison.py

```diff
@@ -0,0 +1,127 @@
+tractograms from different algorithms or parameters need to be compared.
+
+A similar script (scil_bundle_pairwise_comparison.py) is available for bundles,
+with metrics more adapted to bundles (and spatial agreement).
+
+The difference is computed in terms of
+- A voxel-wise spatial distance between streamlines crossing each voxel.
+    This can help to see if both tractography reconstructions at each voxel
+    look similar (out_diff.nii.gz)
+- An angular correlation (ACC) between streamline orientation from TODI.
+    This compares the local orientation of streamlines at each voxel
+    (out_acc.nii.gz)
+- A patch-wise correlation between streamline density maps from both
+    tractograms. This compares where the high/low density regions agree or not
+    (out_corr.nii.gz)
+- A heatmap combining all the previous metrics using an harmonic means of the
+    normalized metrics to summarize general agreement (out_heatmap.nii.gz)
+"""
+
+import argparse
+import logging
+import os
+
+import nibabel as nib
+
+from scilpy.io.utils import (add_overwrite_arg, add_reference_arg,
+                             assert_inputs_exist,
+                             assert_outputs_exist,
+                             assert_output_dirs_exist_and_empty,
+                             add_processes_arg,
+                             add_verbose_arg,
+                             is_header_compatible_multiple_files,
+                             load_tractogram_with_reference,
+                             validate_nbr_processes)
+from scilpy.tractograms.tractogram_operations import tractogram_pairwise_comparison
+
+
+def _build_arg_parser():
+
+    p = argparse.ArgumentParser(
+        description=__doc__,
+        formatter_class=argparse.RawTextHelpFormatter)
+
+    p.add_argument('in_tractogram_1',
+                   help='Input tractogram 1.')
+    p.add_argument('in_tractogram_2',
+                   help='Input tractogram 2.')
+
+    p.add_argument('--out_dir', default='',
+                   help='Directory where all output files will be saved. '
+                        '\nIf not specified, outputs will be saved in the '
+                        'current directory.')
+    p.add_argument('--out_prefix', default='out',
+                   help='Prefix for output files. Useful for distinguishing '
+                        'between different runs [%(default)s].')
+
+    p.add_argument('--in_mask', metavar='IN_FILE',
+                   help='Optional input mask.')
+    p.add_argument('--skip_streamlines_distance', action='store_true',
+                   help='Skip computation of the spatial distance between '
+                        'streamlines.')
+    add_processes_arg(p)
+    add_reference_arg(p)
+    add_verbose_arg(p)
+    add_overwrite_arg(p)
+
+    return p
+
+
+def main():
+    parser = _build_arg_parser()
+    args = parser.parse_args()
+
+    if args.verbose:
+        logging.basicConfig(level=logging.INFO)
```

Comment:
Change this with: 
`logging.getLogger().setLevel(logging.getLevelName(args.verbose))`